fakestorage/object_test.go

```diff
@@ -186,6 +186,14 @@ func checkObjectAttrs(testObj Object, attrs *storage.ObjectAttrs, t *testing.T)
 			t.Errorf("wrong MetaHeader returned\nwant %s\ngot %v", testObj.Metadata["MetaHeader"], val)
 		}
 	}
+	externalUrl := "" // We don't set any `externalURL` value during tests.
```

Comment:
super nit, but can you call this (and others) `externalURL`?

(I thought there was a vet rule for this in Go, but I guess I'm wrong)

Reply:
Sure thing. Thanks for pointing it out; I wasn't aware.

Reply:
Addressed in 97841f3.